docs/tutorials/abn/abn.md

```diff
@@ -62,22 +79,21 @@ Deploy both the frontend and backend components of the application as described
     The frontend service is implemented to call **Lookup()** before each call to the backend service. It sends its request to the recommended backend service.
 
 === "backend"
-    Deploy the *v1* version of the *backend* component as track *default*.
+    Deploy version *v1* of the *backend* component as track *backend*.
 
     ```shell
-    kubectl create deployment backend --image=iter8/abn-sample-backend:latest
-    kubectl expose deployment backend --name=backend --port=8091
-
-    kubectl label deployment backend app.kubernetes.io/name=backend
+    kubectl create deployment backend --image=iter8/abn-sample-backend:v1
     kubectl label deployment backend app.kubernetes.io/version=v1
-    kubectl label deployment backend iter8.tools/track=default
-    kubectl label deployment backend iter8.tools/abn=true
+
+    kubectl expose deployment backend --name=backend --port=8091
     ```
- 
+
+Before calling the backend, the frontend uses Lookup() to identify the track to send requests to. Since there is only one version of the backend deployed, all requests will be sent to it.
```

Comment:
```suggestion
Before calling the backend, the frontend uses `Lookup()` to identify the track to send requests to. Since there is only one version of the backend deployed, all requests will be sent to it.
```